app/scripts/kunden/detail/kundendetail.controller.js

```diff
@@ -101,7 +101,8 @@ angular.module('openolitor-admin')
             if (!$scope.kunde.kontoDaten)
             $scope.kunde.kontoDaten = {
                 nameAccountHolder : $scope.kunde.bezeichnung,
-                addressAccountHolder : $scope.kunde.hausNummer + " " + $scope.kunde.strasse + ", " + $scope.kunde.plz + " " + $scope.kunde.ort
+                addressAccountHolder : $scope.kunde.hausNummer + " " + $scope.kunde.strasse + ", " + $scope.kunde.plz + " " + $scope.kunde.ort,
```

Comment:
Is it really first the hausNummer and then the strasse? This is never done this way in Switzerland and german speaking countries.

Reply:
There is an issue for changing this order. https://github.com/OpenOlitor/OpenOlitor/issues/41